### PR TITLE
New module serviceguard_facts

### PR DESCRIPTION
**Pull request by sandrich**
##### SUMMARY

New facts module to collect HP ServiceGuard cluster facts. This is the first out of 3 modules coming.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
serviceguard_facts

##### ADDITIONAL INFORMATION

```paste below
ansible 2.7.1
  config file = None
  configured module search path = ['/Users/chris/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.7.1/libexec/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.0 (default, Oct  2 2018, 09:18:58) [Clang 10.0.0 (clang-1000.11.45.2)]
```

### CODE REVIEW

**Comment by sandrich** [comment]
bot_status

**Comment by sandrich** [comment]
Anything I can do to speed up the approval process?

File: lib/ansible/module_utils/serviceguard.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import re
```
Comment (resmo) [inline lib/ansible/module_utils/serviceguard.py:28]: missing license header (must not be gpl here)

File: lib/ansible/modules/clustering/serviceguard_facts.py [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: serviceguard_facts
+
+short_description: HP ServiceGuard facts module
+
+version_added: "2.8"
```
Comment (resmo) [inline lib/ansible/modules/clustering/serviceguard_facts.py]: please update verision to 2.9

**Review by resmo — CHANGES_REQUESTED** [review]

**Comment by sandrich** [comment]
@resmo Created a new one #56358

**Pull request closed without merging**